### PR TITLE
Adding click events using useState hook

### PR DESCRIPTION
**Pull request by duskotmane**
# In this PR I added : 
- [ ] A copy of calculate.js and operate.js to a new folder called logic.  
- [ ] big.js library to handle calculations.
- [ ] A function to handle the clicks.
- [ ] propType library to handle props.

### CODE REVIEW

File: src/components/buttons.js [hunk 1]
```diff
@@ -1,29 +1,77 @@
...
+      </button>
+      <button className="button" type="button" onClick={() => handleClick('+/-')}>
+        +/-
+      </button>
+      <button className="button" type="button" onClick={() => handleClick('%')}>
+        %
+      </button>
+      <button className="button operator" type="button" onClick={() => handleClick('/')}>
+        /
+      </button>
```
Comment (omar-muhamad) [inline src/components/buttons.js]: - [ ] When I try to make a divide operation the error in the image below is raised, Please fix that by replacing the slash `/` with the divide symbol `÷`.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @otmaneechchafyky,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/buttons.js (hunk 1 above)
Comment (duskotmane) [reply src/components/buttons.js]: Okay, I am on it. I just forgot to modify it.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @otmaneechchafyky,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by duskotmane** [comment]
This is the first time getting the same reviewer twice 🤣

**Pull request merged**